app/src/main/java/de/ur/explure/model/rating/Rating.kt

```diff
@@ -1,14 +1,18 @@
 package de.ur.explure.model.rating
 
 import android.os.Parcelable
+import com.google.firebase.firestore.DocumentId
+import com.google.firebase.firestore.ServerTimestamp
 import java.util.Date
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
 data class Rating(
-    val id: String,
-    val ratingValue: Int,
-    val createdAt: Date,
-    val routeId: String,
-    val authorId: String,
+    @DocumentId
+    val id: String = "",
+    val ratingValue: Int = 0,
+    @ServerTimestamp
+    val createdAt: Date = Date(),
```

Comment:
Kann Firebase den ServerTimeStamp einfach in ein Date umwandeln? Bzw. brauchen wir das Date() überhaupt? Es wird ja eh auch wenns null ist automatisch mit dem ServerTimeStamp initialisiert, sobald das in Firebase angelegt wird. Oder überseh ich was?

Reply:
1. In meinen tests hat die Umwandlung geklappt
2. Das Date() wird für den leeren Konstruktor gebraucht

Reply:
Also statt Date würde einfach ```:Date? = null``` auch gehen, wenn es nur um den leeren Konstruktor geht. Aber wenns eh klappt, dann hat sich das erledigt.